src/sidebar/components/ToastMessages.tsx

```diff
@@ -16,10 +18,23 @@ function ToastMessages({ toastMessenger }: ToastMessageProps) {
   const messages = store.getToastMessages();
 
   return (
-    <BaseToastMessages
-      messages={messages}
-      onMessageDismiss={(id: string) => toastMessenger.dismiss(id)}
-    />
+    <div
+      className={classnames(
+        // Set an aggressive z-index as we want to ensure toast messages are
+        // rendered above other content
+        'z-10',
+        'absolute left-0 w-full',
+      )}
+    >
+      <BaseToastMessages
+        messages={messages}
+        onMessageDismiss={(id: string) => toastMessenger.dismiss(id)}
+        transitionClasses={{
+          transitionIn:
+            'motion-safe:animate-slide-in-from-right lg:animate-fade-in motion-reduce:animate-fade-in',
```

Comment:
These classes are different from the ones used in via. See https://github.com/hypothesis/via/pull/1182/files#diff-920b0d4ce3a70733a52d88482b0ba52ecb06a64100eb0627953ae857bc913698R471